### ISSUE TITLE
about the training parameters?

### ISSUE THREAD

**Issue by zklgame** [CVLAB-Unibo/Real-time-self-adaptive-deep-stereo#14]
Hi, thank you for sharing your code, it is really a great work!
I used your pretrained weights and it worked well on my own dataset after a few steps to finetune on. Then I wanted to reproduce the results from the very beginning, using the same synthetic dataset and KITTI dataset, but I found it lose 2% if I trained from random initialization.   
So I am curious to know if you can tell me more about the training parameters for training on synthetic dataset and  KITTI dataset separately?
Looking forward to your reply~

**Comment by AlessioTonioni** [comment]
You can find the parameters for the synthetic dataset on the paper.
As for the KITTI tuned weights, if I remember well we fine-tuned the network computing the loss function only on the final prediction and reducing the learning rate to 1E-5.

**Comment by mattpoggi** [comment]
Hi @zklgame,
from our supplementary material: "we perform 1200000 training iterations on the FlyingThings3D dataset using Adam Optimizer and a learning rate of 0.0001, halved after 400k steps and further every 200k until convergence."

**Comment by zklgame** [comment]
I first used the unsupervised method on FlyingThings3D dataset, then used the supervised method on Kitti dataset, and finally reproduced the best result! And I also found that using pure unsupervised or pure supervised method will harm the best result.
